### PR TITLE
Feat/apiserver

### PR DESCRIPTION
**Pull request by panzhen6668**
add apiserver for web  https://github.com/kubesphere/kubeeye/issues/142

/cc @pixiake @ruiyaoOps  @zheng1

### CODE REVIEW

**Comment by zheng1** [comment]
It is not recommended to put vendor in the committed code

File: pkg/utils/sets/byte.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
+/*
```
Comment (zheng1) [inline pkg/utils/sets/byte.go:1]: If these codes are not used for the moment, you can delete them first

File: pkg/apiserver/request/requestinfo.go [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+//
+// Special verbs without subresources:
+// /api/{version}/proxy/{resource}/{resourceName}
+// /api/{version}/proxy/namespaces/{namespace}/{resource}/{resourceName}
+//
+// Special verbs with subresources:
+// /api/{version}/watch/{resource}
+// /api/{version}/watch/namespaces/{namespace}/{resource}
+
+func (r *RequestInfoFactory) NewRequestInfo(req *http.Request) (*RequestInfo, error) {
```
Comment (zheng1) [inline pkg/apiserver/request/requestinfo.go:113]: This part of the logic should not be used, for now, so you can remove it.

**Comment by zheng1** [comment]
I see that there are a lot of code that will not be used at the moment, can we remove it for now?

**Review by zheng1 — CHANGES_REQUESTED** [review]

**Comment by panzhen6668** [comment]
Resubmit  a Pr

**Comment by panzhen6668** [comment]
Resubmit a Pr

**Comment by panzhen6668** [comment]
add apiserver for web  https://github.com/kubesphere/kubeeye/issues/142 
simplify apiserver
/cc @pixiake @ruiyaoOps  @zheng1

**Comment by pixiake** [comment]
@all-contributors please add @panzhen6668  for code

**Pull request merged**